### PR TITLE
Feature/gal 695/trigger episode record creation

### PR DESCRIPTION
**Pull request by zainm97**
## Description
This PR add functionality to flag the person selected to be invited from the LSOA table to be invited once the 'Generate invitations' button is clicked. It also updates the range selected and target fill to percentage the user has chosen to the Phlebotomy database

https://nhsd-jira.digital.nhs.uk/browse/GAL-695

## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: scripts/pipeline/nonprod_population_load/nonprod_population_load.py
Comment (zainm97) [inline scripts/pipeline/nonprod_population_load/nonprod_population_load.py:1]: only added identified_to_be_invited value. Rest is linting

File: terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+  try {
+    let personUpdated = false;
+    let siteUpdated = false;
+
+    const responsePopulation = await updatePersonsToBeInvited(personIdentifiedArray, client)
+    const responsePhlebotomySite = await updateClinicFields(clinicInfo, client)
+
+    if (responsePopulation.every(element => element.value === 200)){
+      console.log("All persons successfully updated")
```
Comment (Amar-NHS) [inline terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js:31]: maybe having the value "200" as named constant for better readability.

**Review by Amar-NHS — COMMENTED** [review]
Looks good to me, good stuff. just 1 comment to address.

File: scripts/pipeline/nonprod_population_load/nonprod_population_load.py [hunk 2]
```diff
@@ -123,8 +186,6 @@ def batch_write_to_dynamodb(lsoa_data):
     male_file = os.getcwd() + file_input_path_1
     female_file = os.getcwd() + file_input_path_2
 
-    ENVIRONMENT = os.getenv("environment")
```
Comment (Abdul-NHS) [inline scripts/pipeline/nonprod_population_load/nonprod_population_load.py:126]: I believe we need these to introduce the env prefix to the table definitions

Comment (zainm97) [reply scripts/pipeline/nonprod_population_load/nonprod_population_load.py:126]: popped it to the top of the file

File: terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js (hunk 1 above)
Comment (zainm97) [reply terraform/src/generateInvitesTriggerLambda/generateInvitesTriggerLambda.js:31]: good shout

**Review by Abdul-NHS — APPROVED** [review]
lgtm

**Review by Amar-NHS — APPROVED** [review]
Approved

**Pull request merged**